frontend/src/locales/en.json

```diff
@@ -317,6 +358,7 @@
       "continueToFxa": "Enter your email above to continue to Mozilla Accounts"
     },
     "nameIsInvitingYou": "{name} is inviting you",
+    "ownerNeedsToConfirmBooking": "If this option is active, appointments are only booked until the calendar owner confirmed it.",
```

Comment:
Please correct my bad English if necessary. In this context it might be even better to say "until you confirmed them", because it's displayed only for the one editing their schedule.